NUNIT_TESTS/UnitTest1.cs

```diff
@@ -0,0 +1,65 @@
+using DALK.PL_ANALYZER.Models.Matches;
+using NUnit.Framework;
+
+namespace Tests
+{
+    public class DalkMatchTests
+    {
+        [TestCase(2, 4)]
+        [TestCase(1, 2)]
+        [TestCase(0, 1)]
+        [TestCase(8, 256)]
+        public void DuelsInPlayOffStageCounterTest(byte howFar, int duelsExpected)
+        {
+            DuelsInPlayOffStageCounter dc = new DuelsInPlayOffStageCounter(howFar);
+            Assert.AreEqual(duelsExpected, dc.HowManyDuels());
+        }
+        [TestCase(2, "Playoffy - �wier�fina�")]
+        [TestCase(1, "Playoffy - P�fina�")]
+        [TestCase(0, "Playoffy - Fina�")]
+        [TestCase(7, "Playoffy - 1/128")]
+        public void PlayOffStageTest(byte howFar, string nameExpected)
+        {
+            PlayOffStage pS = new PlayOffStage(howFar);
+            Assert.AreEqual(nameExpected, pS.GetStageName());
+        }
+        [TestCase(2018, true, "2018 Letnia")]
+        [TestCase(2010, false, "2010")]
+        public void SeasonOneYearTest(int FirstYear, bool IsSummerSeason, string nameExpected)
+        {
+            Season s = new Season() { FirstYear = FirstYear, IsSummerSeason = IsSummerSeason};
+            Assert.AreEqual(nameExpected, s.GetName());
+        }
+
+        [TestCase(2018, 2019, true, "2018/2019 Letnia")]
+        [TestCase(2010, 2011, false, "2010/2011")]
+        public void SeasonTwoYearsTest(int FirstYear, int SecondYear, bool IsSummerSeason, string nameExpected)
```

Comment:
What are you testing here? 
Name you methods with
`Given_When_Then`
Or at least like `When_Click_Then_Action`